avm/res/web/serverfarm/tests/e2e/max/dependencies.bicep

```diff
@@ -6,7 +6,7 @@ param managedIdentityName string
 
 resource managedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2018-11-30' = {
   name: managedIdentityName
-  location: location
+  location: 'eastus'
```

Comment:
why? Does it have to be in the same region as the server farm?

Reply:
Surely keeping all of the resources together would be best for now so we can unblock the CI Env?

Reply:
in theory yes - but if we start also changing the dependency files, cleaning this up later will become a lot harder (also harder to troubleshoot if we overlook a location).

Reply:
In that case I'd rather suggest to hardcode it in the invocating deployment - i.e. in the `main.test.bicep`. Then the workaround is limited to one location

Reply:
updated